src/routes/userRoutes.js

```diff
@@ -90,6 +90,7 @@ router.post('/signup', signupValidator, UserController.signup);
  *             description: server error.
  * */
 router.post('/login', signinValidator, passportCheck, UserController.signin);
+
```

Comment:
What did you change here, kindly avoid making your PR so noisy, Make sure when you touch the file you are going to make a valid change.